### PR TITLE
Add to support to create an SP with role Owner or Reader

### LINKED ISSUE #121: Improve the create the SP with roles as args

### PR DESCRIPTION
**Pull request by uisso**
Close #121

### CODE REVIEW

**Comment by uisso** [comment]
Dudes, useful to create SP with role Owner or Reader

File: tools/azure/createsp.sh [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
+	for entry in $ROLES {
+		if $entry[0] == $role {
+			return $entry[1]
+		}
+	}
+
+	return "false"
+}
+
+if len($ARGS) != "5" {
```
Comment (tiago4orion) [inline tools/azure/createsp.sh]: you can make this change optional. Maybe if not set default to 'owner'.

```sh
defRole = "Owner"
arglen <= len($ARGS)
_, cond <= test $arglen -lt 4
if $cond == "0" {
    echo "usage: "+$ARGS[0]+" <subscription_id> <service_principal> <password> [<role>]"
     exit
}

subscription_id   = $ARGS[1]
service_principal = $ARGS[2]
password          = $ARGS[3]
role = $defRole
if $arglen == "5" {
    role = $ARGS[4]
}
```

Comment (katcipis) [inline tools/azure/createsp.sh]: If the new parameter is obligatory the docs should also be updated, I think we mention this tool there =)

**Comment by lborguetti** [comment]
@uisso :boom:

**Comment by uisso** [comment]
Hey mates @tiago4orion @katcipis @lborguetti , done! Sorry for my late reply. Some else?

**Comment by katcipis** [comment]
LGTM

**Pull request merged**